src/rgw/rgw_sal_motr.cc

```diff
@@ -2303,13 +2312,13 @@ int MotrObject::get_bucket_dir_ent(const DoutPrefixProvider *dpp, rgw_bucket_dir
 int MotrObject::update_version_entries(const DoutPrefixProvider *dpp)
```

Comment:
Do we still need this function?

Reply:
Yes, to update previous version fields we require this function.